src/commands/transaction/accountlink.ts

```diff
@@ -36,6 +38,13 @@ export class CommandOptions extends AnnounceTransactionsOptions {
         validator: new BinaryValidator(),
     })
     action: number;
+
+    @option({
```

Comment:
can this be pushed inside `ProfileCommand` as a parameter that is always needed for profile-specific commands?

Reply:
+1

Reply:
argh... knowing that since the beginning would have saved me time :p
pretty cool